<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdDetails</class>
 <widget class="QWidget" name="CmdDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CmdDetails</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="argDetailsLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Command</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="nameLabel">
              <property name="text">
               <string>name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="commentLabel">
              <property name="text">
               <string>comment</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="commentEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="n_argsLabel">
              <property name="text">
               <string>n_args</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="n_argsEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="idLabel">
              <property name="text">
               <string>id</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="idEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="_CmdArgContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>374</width>
               <height>181</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="CmdArgContents"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addArgButton">
            <property name="text">
             <string>Add Arg</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="deleteCmdButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
